### PR TITLE
NXP-28434: Rely export data on multi picture views

### PR DESCRIPTION
**Pull request by vpasquier**

### CODE REVIEW

File: nuxeo-ai-pipes/src/main/java/org/nuxeo/ai/pipes/functions/PropertyUtils.java [hunk 1]
```diff
@@ -202,13 +205,33 @@ public static BlobTextFromDocument docSerialize(DocumentModel doc, Set<String> p
...
+                    propertiesList));
         }
 
         return blobTextFromDoc;
     }
 
+    /**
+     * Get the best conversion of the original blob if exist. If not return the original blob.
+     */
+    private static ManagedBlob getPictureConversion(DocumentModel doc, ManagedBlob originalContent) {
```
Comment (PedroCardoso) [inline nuxeo-ai-pipes/src/main/java/org/nuxeo/ai/pipes/functions/PropertyUtils.java]: Do we have any other ticket to check that the conversion is connected to the final image, and not a previous one ?

Comment (vpasquier) [reply nuxeo-ai-pipes/src/main/java/org/nuxeo/ai/pipes/functions/PropertyUtils.java]: There is a ticket for that to do later, but apparently, the code i set shoudl be already safe

Comment (vpasquier) [reply nuxeo-ai-pipes/src/main/java/org/nuxeo/ai/pipes/functions/PropertyUtils.java]: https://jira.nuxeo.com/browse/NXP-28435

File: nuxeo-ai-pipes/src/main/java/org/nuxeo/ai/pipes/functions/PropertyUtils.java [hunk 2]
```diff
@@ -202,13 +205,33 @@ public static BlobTextFromDocument docSerialize(DocumentModel doc, Set<String> p
...
+        MultiviewPicture managedConversion = doc.getAdapter(MultiviewPicture.class);
+        // Check if multiview picture adapter can be accessible
+        if (managedConversion == null) {
+            return originalContent;
+        }
+        Optional<PictureView> pictureView = Arrays.stream(managedConversion.getViews())
+                                                  .filter(view -> view.getWidth() >= 299 && view.getHeight() >= 299)
```
Comment (PedroCardoso) [inline nuxeo-ai-pipes/src/main/java/org/nuxeo/ai/pipes/functions/PropertyUtils.java]: Put these as constants :
MIN_REQUIRED_WIDTH,  MIN_REQUIRED_HEIGHT

**Pull request merged**